files/opensesame.spec

```
@@ -126,3 +126,14 @@ description=How often to get new data (default: every 60 seconds, which is the h
 default = 6000
 type = unsigned_short
 
+[weatherstation/enable]
+description = enables/disables weatherstation
+type = boolean
+default = 0
+
+[opensensemap/box_id]
```

Probably put below weatherstation or do you think there is something else to publish? (I don't think so.)

```suggestion
[weatherstation/opensensemap/id]
```